tools/gromacs/minim.xml

```diff
@@ -18,7 +18,7 @@
         ln -s '$top_input' ./top_input.top &&
   
         gmx grompp -f ./minim.mdp -c ./solv_ions.gro -r ./solv_ions.gro -p ./top_input.top -o em.tpr &>> verbose.txt &&
-        gmx mdrun -deffnm em &>> verbose.txt
+        gmx mdrun -nt ${GALAXY_SLOTS:-4} -deffnm em &>> verbose.txt
```

Comment:
@bgruening, don't really know whether 4 is a good choice here.

Reply:
I think this is ok!

Reply:
can we add this to @tsenapathi's bash script as well? e.g. here: https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/blob/master/tools/free_energy/alchemical_run/gmx_fep.sh#L50

Reply:
We should! GALAXY_SLOTS is a real ENV variable so it should also work in the bash script.

Reply:
cool